prime-router/src/main/kotlin/ActionMessages.kt

```diff
@@ -1,13 +1,55 @@
 package gov.cdc.prime.router
 
+/**
+ * Possible error codes when parsing/processing messages. Includes Hl7, CSV, FHIR, et al.
+ */
+enum class ErrorCode {
+    INVALID_MSG_PARSE_GENERAL,
```

Comment:
Created a INVALID_MSG_PARSE_X for each data type defined in Element.kt. This way these codes are not specific to just HL7